### DISCUSSION TITLE
AddMotfiAnnotations for cisbp and JASPAR at the same time

### DISCUSSION THREAD

**Discussion by tzhu-bio** [GreenleafLab/ArchR#1039] [General]
Is it possible to add cisbp and JASPAR at the same time in **AddMotfiAnnotations**?

Or can I run **peakAnnoEnrichment** with cisbp and JASPAR at the same time after I add them in separately.

**Comment by rcorces** [comment]
As indicated in the function documentation, you can provide a custom set of motifPWMs. So you can do whatever you want.
https://www.archrproject.com/reference/addMotifAnnotations.html

But ArchR does not provide any support for combining two separate motif annotations. You would need to create a combined set of motifPWMs before adding the motif annotations.
